ui/src/react-admin/modules/shared/components/ContentPicker/helpers/parse-picker.ts

```diff
@@ -38,7 +38,7 @@ export const parseSearchQuery = (input: string): string => {
 			type: ToastType.ERROR,
 		});
 
-		return 'Ongeldige zoekfilter';
```

Comment:
We kunnen ook een vertaling toevoegen:
```
AdminConfigManager.getConfig().services.i18n.tText(
				'Ongeldige zoekfilter'
			)
```

en dan npm run extract-translations runnen

Reply:
Zie https://github.com/viaacode/avo2-client/pull/1951#discussion_r1498877294

Reply:
zie https://github.com/viaacode/avo2-client/pull/1951#discussion_r1499328888